package.json

```diff
@@ -11,12 +11,19 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@prisma/client": "^5.20.0",
     "@clerk/nextjs": "^5.7.1",
+    "@prisma/client": "^5.20.0",
+    "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-slot": "^1.1.0",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
     "lenis": "^1.1.13",
+    "lucide-react": "^0.447.0",
```

Comment:
Why are we using two icon libraries. Can it be one either radix ui icon or lucide-react ?

Reply:
lucide-react is the one that shadcn recommends

Reply:
We are not using any icons on the home page, so we can remove this lucide-react icon.

Reply:
Ah I see, leave the package as it is . Other pr need the package